### ISSUE TITLE
Is possible to run the server on a complete separate computer without Stardew valley installed?

### ISSUE THREAD

**Issue by KEKKO96** [stardew-valley-dedicated-server/server#11]
Hi, I have an old computer with Ubuntu and Docker that I use as my personal server. I would like to know if it is possible to host a Stardew Valley server on it, something similar to Minecraft dedicated server, without having the game installed. Thanks!

**Comment by JulianVallee** [comment]
Yes, the Docker container downloads the game files for the server automatically, so no need to manually install the game.

However, you will need a Steam account with 2FA disabled for the automatic download. Once downloaded, the steam account is no longer needed and could be removed/secured with 2FA again. 

And yup, it's a dedicated standalone server similar to Minecraft servers - but please keep in mind that it's currently in alpha release (need to update the readme) and thus comes with quite a few bugs.

PS: It might be heavy on the CPU since it still runs the full game under the hood, so quite curious to hear about your experience on an older machine!

**Comment by KEKKO96** [comment]
I tried it last night with a friend and had no problems, even though the host is an old laptop with a Pentium 987 and 4GB of RAM, we experienced no problems or lag. It's a little heavy on the CPU, in fact I can hear the fan running very loudly, but it doesn't seem to be a problem.
However I am having some problems with the mods, I added the volume "./mods:/data/Stardew/Mods/extra" in the compose file and it created a folder called "mods" where I put all the folders with the mods but it seems like they don't load/work. What can I do about this? Thanks.

**Comment by JulianVallee** [comment]
Thanks for the feedback - sounds way better than expected! 🙏 

Currently, the extra mods volume should be `./mods:/data/Mods/extra`. Support is still work-in-progress, so this is subject to change. That said, I'm personally using it to add `ModConfigMenu` and `TimeSpeed` for development, so it should work at least
